.kube/app/Dockerfile

```diff
@@ -36,9 +38,30 @@ RUN apt-get install -y \
 RUN docker-php-ext-configure gd --with-freetype --with-jpeg
 RUN docker-php-ext-install -j$(nproc) gd pdo_mysql mbstring exif pcntl bcmath bz2 zip soap intl opcache
 
-RUN apt-get install -y libmagickwand-dev --no-install-recommends
+RUN apt-get install -y \
+    libmagickwand-dev \
+    libmagickcore-dev \
+    --no-install-recommends
 
-RUN pecl install imagick
+# START BUILDING IMAGICK FROM SOURCE - this can be removed once pecl fixes issues installing with php8.4
+# TODO - this can be removed when https://github.com/Imagick/imagick/issues/698 is resolved
+RUN apt-get install -y \
+    $PHPIZE_DEPS \
+    libtool
+
+RUN mkdir -p /tmp/imagick
+WORKDIR /tmp/imagick
+RUN curl -L -o /tmp/imagick.tar.gz https://github.com/Imagick/imagick/archive/tags/${IMAGICK_VERSION}.tar.gz
```

Comment:
When I tried to build the container I got the error 

```
failed to solve: process "/bin/sh -c curl -L -o /tmp/imagick.tar.gz https://github.com/Imagick/imagick/archive/tags/${IMAGICK_VERSION}.tar.gz" did not complete successfully: exit code: 6
```

Reply:
When abstracting version to an environment variable it broke the curl command reverted to static url.